lib/Compiler.js

```diff
@@ -59,13 +64,29 @@ function compileSolc(logger, contractFiles, cb) {
           compiled_object[className].functionHashes = contract.hashes;
           compiled_object[className].abiDefinition = JSON.parse(contract.abi);
           compiled_object[className].filename = fileName;
+
+          
         }
 
         fileCb();
       });
     },
     function (err) {
-      cb(err, compiled_object);
+      cb(err, compiled_object);  
+      
+      embark.events.on("outputDone", function() {
+        if(outputBinary){
+          Object.keys(compiled_object).map(function(className, index) {
+            fs.writeFile(outputDir + className + ".bin", compiled_object[className].bin, (err) => {
+              if (err) {
+                logger.error("Error writing binary file: " + JSON.stringify(err));
+              }
+            });
+          });
+        }
+      });
+
+
```

Comment:
Double newline here too.

Reply:
Yeah... I'll set up my ESlint to warn me about those multiple empty lines. It's a bad habit I've picked up.